Tasks/ServiceFabricDeploy/task.json

```diff
@@ -270,6 +270,15 @@
             "required": false,
             "groupname": "upgrade",
             "visibleRule": "overridePublishProfileSettings = true && isUpgrade = true && upgradeMode = Monitored"
+        },
+        {
+            "name": "skipUpgradeSameTypeAndVersion",
+            "type": "boolean",
+            "label": "Skip upgrade for same Type and Version",
+            "defaultValue": "false",
+            "required": false,
+            "groupname": "upgrade",
+            "helpMarkDown": "Indicates whether an upgrade will be skipped if the same application type and version already exists in the cluster, otherwise the upgrade fails during validation. If enabled, re-deployments are idempotent."
```

Comment:
Sure, I placed it in the upgrade group since it was related to upgrades, but yes it looks and behaves a bit odd, when selecting/deselecting the override publish settings and upgrade checkbox.